packages/fast-components/src/progress/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./base-progress";
```

Comment:
should we export this?

Reply:
Is there a reason not to? I don't see the harm in doing so. This way someone could build their own custom element off of base progress.